lib/LaTeXML/resources/XSLT/LaTeXML-structure-xhtml.xsl

```diff
@@ -593,6 +595,27 @@
     </xsl:element>
   </xsl:template>
 
+  <xsl:template match="ltx:contact[@role='orcid']">
+    <xsl:param name="context"/>
+    <xsl:text>&#x0A;</xsl:text>
+    <xsl:element name="span" namespace="{$html_ns}">
+      <xsl:variable name="innercontext" select="'inline'"/><!-- override -->
+      <xsl:call-template name="add_id"/>
+      <xsl:call-template name="add_attributes">
+      </xsl:call-template>
+      <xsl:apply-templates select="." mode="begin">
+        <xsl:with-param name="context" select="$innercontext"/>
+      </xsl:apply-templates>
+      <xsl:apply-templates>
+        <xsl:with-param name="context" select="$innercontext"/>
+      </xsl:apply-templates>
+      <xsl:apply-templates select="." mode="end">
+        <xsl:with-param name="context" select="$innercontext"/>
+      </xsl:apply-templates>
+    </xsl:element>
+    <xsl:text>&#x0A;</xsl:text>
+  </xsl:template>
```

Comment:
this copy-pasta was rather bothersome for me to create... I wonder if we could get a "Generic" contact rule that simply copies over the content and add *ALL* generics under it via `@role='X' or @role='Y' or...`. With a comment explicitly inviting developers from the future to add more generics to it :> 

I don't know enough about the code here at the moment, so I stayed away from such sweeping changes.